rag_app/query.py

```diff
@@ -0,0 +1,42 @@
+import typer
+import openai
+from rag_app.models import TextChunk
+from lancedb import connect
+import textwrap
+from typing import List
+
+app = typer.Typer()
+
+
+@app.command(help="Query LanceDB for some results")
+def db(db_path: str, table_name: str, query: str, n: int = 3):
+    db = connect(db_path)
```

Comment:
`lancedb` creates a new db by default if the path doesn't exist. Do we want to support this behaviour or throw an error? If the db doesn't exist then the query will fail since the table won't exist too in an empty db.

I think throwing an error here might be a wiser choice